Build system upgrades (#406)
ostis-ai/sc-machine: docs/sc-memory/api/cpp/extended/agents/keynodes.md

    This is correct for only versions of sc-machine that >= 0.10.0.
--- 

> The **C++ Keynodes API** allows developers to work with key nodes in the knowledge graph that represent significant concepts or entities. This documentation describes how to create, retrieve, and manipulate keynodes within the sc-machine, enabling efficient management of critical data points in semantic networks.

Reviewer: ```suggestion
The **C++ Keynodes API** allows developers to work with key //it can be a node and it can be a connector\\ in the knowledge graph that represent significant entities. This documentation describes how to create, retrieve, and manipulate keynodes within the sc-machine, enabling efficient management of critical data points in semantic networks.
```
